infra/examples-dev/aws-msft-365/main.tf

```diff
@@ -41,10 +41,14 @@ locals {
   bulk_sources = {
     "hris" = {
       source_kind = "hris"
+      rules = {
+        columnsToRedact = []
+        columnsToPseudonymize = [
+          "email",
```

Comment:
I'd clarify somehow if when adding here rules, we are overriding the rules created or if they are going to be concatenated to the existing ones